app/js/controller/list_controller.js

```diff
@@ -26,6 +26,14 @@ export default class ListController extends Controller {
     this.refreshInstalledList();
   }
 
+  showAlertDialog(msg) {
+    if (!this.alertDialog) {
```

Comment:
You're not making very heavy use of private ```_``` tags in vars. I think it would be better to use these.